samples/java/README-ptbr.md

```diff
@@ -0,0 +1,53 @@
+# Lead Integration - Java
+
+[English version](README.md)
+
+Um simples projeto usando Java para demonstrar como receber leads do zap+.
+
+Por favor, confira nossa [documentação](https://developers.grupozap.com/) para mais informações.
+
+## Recursos Utilizados
+
+- [Java 11](https://openjdk.java.net/projects/jdk/11/)
+- [Maven 3.8.1](https://maven.apache.org/)
+- [Spring Boot 2.5.0](https://spring.io/projects/spring-boot)
+- [Docker](https://www.docker.com/get-started)
```

Comment:
@giovannicaldeira put the lombok in this list

Reply:
@robsonoduarte added lombok.